src/main/webapp/scripts/common.js

```diff
@@ -12,14 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-const scrollTopButton = document.getElementById("to-top-button");
-
 window.onscroll = function() {
-//   if (document.body.scrollTop > 10 || document.documentElement.scrollTop > 10) {
-//     scrollTopButton.style.display = "block";
-//   } else {
-//     scrollTopButton.style.display = "none";
-//   }
```

Comment:
The hiding of the scroll button isn't needed anymore, as the decision has been made to keep it visible at all times, rather than set a scroll amount to reveal the visibility. It was commented out but I removed it as it is not needed anymore.

Reply:
Thanks for the clarification Lorna! Now you can git rid of the whole `window.onscroll` function then as it technically does nothing (only getting the button element from the document without changing its behaviour) :)